### PR TITLE
Ft bus info

### PR DESCRIPTION
**Pull request by JPaul23**

### CODE REVIEW

**Comment by David-Neza** [comment]
This branch has conflicts that have to be resolved

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+```
+
+- `npm run dev`
+
+## Todo
+get welcome page
+`http://localhost:your post/`
+
+>>>>>>> setup project postgres
+>>>>>>> setup project postgres
```
Comment (christianR15c) [inline README.md]: please review this file

**Review by christianR15c — CHANGES_REQUESTED** [review]

File: src/controllers/company.js [hunk 2]
```diff
@@ -45,16 +45,26 @@ class Companies {
...
+    const companyNotFoundResponse = req.t('company_message.id_not_found')
+    return Company
+      .findAll()
+      .then(companies => {
+        if (companies) {
+          return res.status(200).send(companies)
+        } else {
+          return res.status(400).send({
+            message: `${companyNotFoundResponse}`
+=======
```
Comment (christianR15c) [inline src/controllers/company.js]: checkout to this file it has unresolved problems

**Review by christianR15c — CHANGES_REQUESTED** [review]

File: src/controllers/company.js (hunk 2 above)
Comment (JPaul23) [reply src/controllers/company.js]: your changes have been addressed

File: README.md (hunk 1 above)
Comment (JPaul23) [reply README.md]: your changes have been addressed

**Review by christianR15c — APPROVED** [review]

**Pull request merged**